src/components/event/EventDetailInfoCard.tsx

```diff
@@ -23,20 +23,20 @@ import { getEventStatus } from '../../helpers/event';
 import { Chip } from '../Chip';
 import EventBodyCard from './EventBodyCard';
 import { EventAction, EventTreeNode } from '../../models/EventAction';
-import { useRootStore, useSelectedStore } from '../../hooks';
+import { useSelectedStore } from '../../hooks';
 
 interface Props {
 	node: EventTreeNode;
 	event: EventAction | null;
 	childrenCount?: number;
 	children?: React.ReactNode;
+	isEmbedded: boolean;
```

Comment:
make this field optional (use `?` symbol as in case of children and childrenCount above), then remove all `isEmbedded={false}` in other places